deps/spidershim/test/value.cc

```diff
@@ -921,6 +947,361 @@ TEST(SpiderShim, ExternalStringResourceDestructorCalled) {
   EXPECT_TRUE(externalOneByteStringResourceDestructorCalled);
 }
 
+// Begin code copied from THREADED_TEST(StringWrite) in test-api.cc.
```

Comment:
Ah, I missed that the copyright header at the top of the file is "Copyright 2015 the V8 project authors." Ok, that makes sense, although I would then update the header to be "Copyright 2012 - 2015 the V8 project authors." (Or remove the year.)